x-pack/plugins/alerting/server/rules_client/lib/get_alert_from_raw.ts

```diff
@@ -98,20 +99,20 @@ export function getPartialRuleFromRaw<Params extends RuleTypeParams>(
     ...s,
     rRule: {
       ...s.rRule,
-      dtstart: new Date(s.rRule.dtstart),
-      ...(s.rRule.until ? { until: new Date(s.rRule.until) } : {}),
+      dtstart: new Date(s.rRule.dtstart).toISOString(),
+      ...(s.rRule.until ? { until: new Date(s.rRule.until).toISOString() } : {}),
     },
   }));
   const includeSnoozeSchedule =
     snoozeSchedule !== undefined && !isEmpty(snoozeSchedule) && !excludeFromPublicApi;
   const isSnoozedUntil = includeSnoozeSchedule
-    ? calculateIsSnoozedUntil({
+    ? getRuleSnoozeEndTime({
```

Comment:
I do not think that's equivalent before it was a boolean and now it is an object. We should think more about this one.

Reply:
@XavierM  I realized we casted this object as `PartialRule<Params>` but the type definition expects as `isSnoozedUntil?: Date | null;` instead of `string`. I made the fix and everything seemed fine. Is it the other way around that it should be? and we need to fix the `Rule` type in TypeScript to expect a string?